qiskit/circuit/library/standard_gates/equivalence_library.py

```diff
@@ -384,6 +410,18 @@
 u3_qasm_def.rz(phi+3*pi, 0)
 _sel.add_equivalence(U3Gate(theta, phi, lam), u3_qasm_def)
 
+q = QuantumRegister(1, 'q')
+theta = Parameter('theta')
+phi = Parameter('phi')
+lam = Parameter('lam')
+u3_qasm_def = QuantumCircuit(q)
+u3_qasm_def.rz(lam, 0)
+u3_qasm_def.sxdg(0)
+u3_qasm_def.rz(theta+pi, 0)
+u3_qasm_def.sxdg(0)
```

Comment:
hmm why use sxdg in here rather than sx? The original decomposition has RX(pi/2) in it which should be the same as sx.

I thought you want to use `sxdg` to write new decompositions for `rz_to_rxry` (which uses RX(-pi/2)).

Reply:
Yep, my bad I got confused between the Rz(-pi/2)s and Rx(pi/2). 

But we can add that, as `RZ(theta) = RX(-pi/2) RY(theta) RX(pi/2)`. That'll have a global phase difference to the current RZ but we're waiting for the global phase PR anyways, right?

Reply:
So there are quite a few equivalences using RX(pi/2) and RX(-pi/2), does it make sense to duplicate all of them? Or is it fine to just have the RX->SX/SXdg equivalence? Maybe that's a difficult equivalence to resolve because the transpiler would need to compare the numerical pi/2 value in the rotation angle 🤔  If that were the case then it might make sense to replace all RX(pi/2)/RX(-pi/2) with SX and add SX->RX equivalences.